Libplanet.Tests/KeyStore/ProtectedPrivateKeyTest.cs

```diff
@@ -694,6 +817,16 @@ public void WriteJson()
             ProtectedPrivateKey key = ProtectedPrivateKey.FromJson(json);
             Assert.Equal(AddressFixture, key.Address);
             Assert.Equal(AddressFixture, key.Unprotect(PassphraseFixture).PublicKey.ToAddress());
+
+            using (var stream = new MemoryStream())
+            {
+                Fixture2.WriteJson(stream);
+                json = Encoding.UTF8.GetString(stream.ToArray());
+            }
+
+            ProtectedPrivateKey key2 = ProtectedPrivateKey.FromJson(json);
+            Assert.Equal(AddressFixture, key.Address);
```

Comment:
Shouldn't this be like below?
```suggestion
            Assert.Equal(AddressFixture2, key2.Address);
```